src/sparql/icd11foundation-relevant-signature.sparql

```diff
@@ -1,12 +1,11 @@
-prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 
-### All diseases
```

Comment:
nit: This could be something like all subclasses of "ICD Category" except classes that are also subclasses of "Extension Codes"

Reply:
Once the build is done I can make this change.